src/ontology/mondo-edit.obo

```diff
@@ -332934,13 +332928,20 @@ id: MONDO:0017400
 name: hypoplastic pancreas-intestinal atresia-hypoplastic gallbalder syndrome
 def: "Hypoplastic pancreas-intestinal atresia-hypoplastic gallbladder syndrome is a rare, potentially fatal, genetic, visceral malformation syndrome characterized by neonatal diabetes, hypoplastic or annular pancreas, duodenal and jejunal atresia, as well as gallbladder aplasia or hypoplasia. Patients typically present intrauterine growth restriction, failure to thrive, malnutrition, intestinal malrotation, malabsorption, conjugated hyperbilirubinemia, acholia and infections. Cardiac anomalies may also be associated." [Orphanet:293864]
 subset: ordo_malformation_syndrome {source="Orphanet:293864"}
+synonym: "diabetes, neonatal, with pancreatic hypoplasia, intestinal atresia, and gallbladder aplasia or hypoplasia" RELATED [OMIM:615710]
 synonym: "hypoplastic pancreas-intestinal atresia-hypoplastic gallbladder syndrome" RELATED [Orphanet:293864]
+synonym: "Mitchell-RILEY syndrome" RELATED [OMIM:615710]
+synonym: "Mitchell-Riley syndrome" EXACT [MONDO:0014315]
+synonym: "Mitchell-Riley syndrome" EXACT [MONDO:Lexical, OMIM:615710]
```

Comment:
these three synonyms above are all the same. Could you please delete Mitchell-RILEY syndrome" RELATED [OMIM:615710] and synonym: "Mitchell-Riley syndrome" EXACT [MONDO:0014315]. The 2nd one is just from the obsoleted term, we don't need to reference the obsoleted Mondo class.